examples/run/cuda/truth_fitting_example_cuda.cpp

```diff
@@ -156,9 +156,12 @@ int main(int argc, char* argv[]) {
         0.01 / detray::unit<scalar>::GeV,
         1 * detray::unit<scalar>::ns};
 
+    // Propagation config
+    detray::propagation::config propagation_config(propagation_opts);
+
     // Fitting algorithm object
     typename traccc::fitting_algorithm<host_fitter_type>::config_type fit_cfg;
-    fit_cfg.propagation = propagation_opts.config;
+    fit_cfg.propagation = propagation_config;
```

Comment:
Same here. Couldn't we just

```suggestion
    fit_cfg.propagation = propagation_opts;
```

?